aws-credentials/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Create a dir
+  file:
+    path: /home/ubuntu/.aws/
+    state: directory
+    owner: ubuntu
+    group: ubuntu
```

Comment:
These should use the same user/group variables that we use in chia-blockchain role, etc (and just default to ubuntu)